doc/source/io.rst

```diff
@@ -4633,6 +4635,46 @@ Read only certain columns of a parquet file.
    os.remove('example_pa.parquet')
    os.remove('example_fp.parquet')
 
+
+Handling Indexes
+''''''''''''''''
+
+Dumping a ``DataFrame`` to parquet may include the implicit index as one or more
+columns in the output file. Thus, this code:
+
+.. ipython:: python
+
+    import pandas
```

Comment:
no need to import pandas here. Pandas is already imported as pd, so you can use that on the line below